packages/nc-gui/composables/useDataReflection.ts

```
@@ -0,0 +1,95 @@
+  const getConnectionDetails = async (baseId?: string) => {
+    if (!activeWorkspace.value?.id) return
+
+    const res = await $api.internal.getOperation(activeWorkspace.value.id, NO_SCOPE, {
+      operation: 'getDataReflection',
+    })
+
+    if (baseId) {
+      selectedBase.value = baseId
+    }
+
+    return (connectionDetails.value = res)
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)